### PR TITLE
Translation of the Apple Pay article.

### PR DESCRIPTION
**Pull request by croath**

### CODE REVIEW

File: 2014-12-8-apple-pay.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
+---
+title: Pay
+author: Jack Flintermann
+category: ""
+translator: Croath Liu
+excerpt: "你要在网上买东西的那一刻你会发现有一种现代化带来的独有的焦虑感。那种感觉不能用语言形容，大概是这样的：\"我的信用卡去哪了？卡号是多少？我好想买这个啊卡去哪了！\""
+---
+
+你要在网上买东西的那一刻你会发现有一种现代化带来的独有的焦虑感。那种感觉不能用语言形容，大概是这样的："我的信用卡去哪了？卡号是多少？我好想买这个啊卡去哪了！"
+
```
Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: How badly do I actually want this thing, anyway?  `this thing` 是不是指的要买的 `something` 这个东西呢？如果直译的话，应该是：“我究竟有多想要买这个东西呢？”。

File: 2014-12-8-apple-pay.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+当你用 iOS 设备的时候，这种困难又加剧了一层：身为运动员或宇航员的你很适合采用一手拿着信用卡一手拿着手机输入的优美姿势。（开个玩笑，但我打赌苹果应该在某个实验室里面这样实验过了。）
```
Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: 运动员 -> 体操运动员吧。如果直译的话是：很有可能你的卡并不在身边，也有可能你会一手拿着信用卡另一手同时在手机上输入，这样的壮举最好还是留给体操运动员和宇航员吧。

File: 2014-12-8-apple-pay.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+如果你刚好在开发一款能够接受信用卡付款的应用，这种状况将直接影响你的收入。
```
Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: this unfortunate phenomenon is directly correlated to lost revenue. 这种不幸的现象直接会导致你收入的减少。

File: 2014-12-8-apple-pay.md (hunk 1 above)
Comment (croath) [inline 2014-12-8-apple-pay.md]: 这里我纠结了很久也没能理解是想说什么。。。

File: 2014-12-8-apple-pay.md (hunk 2 above)
Comment (croath) [inline 2014-12-8-apple-pay.md]: 嗯感觉直译会比较妥当一点

File: 2014-12-8-apple-pay.md (hunk 3 above)
Comment (croath) [inline 2014-12-8-apple-pay.md]: 嗯这里也是，直译比较好。

**Comment by croath** [comment]
@rickytan 帮忙看一眼？

File: 2014-12-8-apple-pay.md (hunk 1 above)
Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: 或者直接问作者？

Comment (croath) [inline 2014-12-8-apple-pay.md]: Hi @jflinter , sorry to bother you but we have a question about your article `Pay` on NSHipster.com .

We're translating your article to Madarin version to post it on NSHipster.cn(as well you could visit nshipster.cn for a look or checkout the `zh-Hans` branch. 

But there's a small problem here: In the first paragraph you wrote `Where is my credit card? What is its number? How badly do I actually want this thing, anyway?`, I don't quite understand what the last sentence mean. Does `this thing` mean the credit card or the goods users wanna buy?

I know it's a not-that-important question around this article, even not the point at any level, but we just wanna make the translation absolutely perfectly right.

Thanks.

**Comment by rickytan** [comment]
这几天在提交论文，流程麻烦……

**Comment by jflinter** [comment]
Hi! So cool that you're translating this, happy to help. I was trying to
make a (mediocre) joke in the first paragraph, like the user's internal
dialogue would be "I don't feel like finding my credit card; so maybe I
don't need to make this purchase after all". So "this thing" would refer to
the goods in this case.

One unrelated thing you may want to highlight more in the article is that
Apple Pay is US-only at the moment (I think I mention it once in passing
already).

Cheers!

**Comment by croath** [comment]
@jflinter Thanks for the quick reply! Got your opinion yet so we can make it right here :)
And yep we'll highlight the US-only words to avoid misunderstanding as you wish.

Nice day :lollipop:

File: 2014-12-8-apple-pay.md (hunk 1 above)
Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: :+1:

**Comment by LiLejia** [comment]
:+1:  Thanks! @jflinter

File: 2014-12-8-apple-pay.md (hunk 1 above)
Comment (croath) [inline 2014-12-8-apple-pay.md]: @LiLejia 所以，这处保持现在这样 OK？

Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: 还是“我究竟有多需要这张信用卡”？来对应之后的 Apple Pay 让你不不需要使用信用卡了？

Comment (croath) [inline 2014-12-8-apple-pay.md]: 呃呃呃，这里原作者说的 this thing 是想买的东西。。。

Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: 啊啊啊，Sorry，记反了，那就是“我究竟有多需要买的这个东西呢？”来对应之后的 Apple Pay 能让“这种不幸的现象直接会导致你收入的减少”。

Comment (croath) [inline 2014-12-8-apple-pay.md]: 嗯，后面那处已经改了。其实 `How badly do I want...` 其实可以直接翻译成 `我好想要...`

Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: `How badly do I actually want this thing, anyway? `
对，我的意思是原文是疑问句，你翻译的意思成了肯定句，翻译完的意思是不是有变？
这里这句话的意思是因为信用卡的繁琐，用户对自己购买的行为产生了怀疑，所以会降低开发者的收入，所以 Apple Pay 很必要，所以这个疑问的语气对于上下文逻辑很重要。

Comment (croath) [inline 2014-12-8-apple-pay.md]: 不同意，这句话的意思就是「我太想要这个东西了」，并不是「我到底有多想要这个东西呢？」，How badly 就是一个肯定句，完全没有疑问的意思。

Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: 但是是问号句子是问号，必须是疑问句啊，这个词决定不了这句话是疑问句，How badly 也可以表疑问语句。@rickytan 您觉得呢？

File: 2014-12-8-apple-pay.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+
+至此，用户可以选择寄送地址以及基于其居住地得到的可用物流方式。用户最终选择的 `shippingAddress` 和 `shippingMethod` 会成为回调到 delegate 方法 `paymentAuthorizationViewController:didAuthorizePayment:completion:` 中 `PKPayment` 对象的某些属性。
+
+> 文中提到的所有源码可以在这个项目中找到：https://github.com/jflinter/ApplePayExample 
+
+* * *
+
+虽然 Apple Pay 只开放了很少的 API，但[很多](https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewFeature?id=927678292&mt=8&ls=1) 应用还是借此实现了多种多样的功能，你可以通过定制化应用到自己的应用中。其实它开启了一个购买的新篇章，比如说，用户根本不用先注册就可以买东西。
+
+随着越来越多的应用开始使用 Apple Pay(也随着越来越多的用户拥有支持 Apple Pay 的设备)，很快这就会成为 iOS 应用中普遍存在的支付方式。我很乐意看一看你们都如何用 Apple Pay 来丰富的你的产品 - 如果你有任何问题，或者想像我展示你的成果，请[联系我](mailto:jack+nshipster@stripe.com)！
```
Comment (rickytan) [inline 2014-12-8-apple-pay.md]: 想 “向” 我  

typo

Comment (croath) [inline 2014-12-8-apple-pay.md]: :lollipop:

File: 2014-12-8-apple-pay.md (hunk 1 above)
Comment (croath) [inline 2014-12-8-apple-pay.md]: Any other ideas here?

Comment (LiLejia) [inline 2014-12-8-apple-pay.md]: Nope... :+1:

**Pull request merged**